critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -29,6 +29,7 @@ def mock_get_entity_by_id(id, type='release_group'):
         }
 
 
+@mock.patch('brainzutils.musicbrainz_db.release.browse_releases', Mock(return_value=None))
```

Comment:
I would prefer to explicitly mock this on all methods where it's needed, instead of at a class level